conf/application.conf

```diff
@@ -173,8 +173,8 @@ feature-switch {
   }
 
   wis008 {
-    enabled = false
-    released-in-production = false
+    enabled = true
+    released-in-production = true
```

Comment:
Are we sure we can make these true when the [target release](https://confluence.tools.tax.service.gov.uk/display/MTE/Feature+Release+Roadmap) for wis008 is "TBC"?

Reply:
Yes these can be true as they don't affect any live environments. The app-configs for the individual environments have been given the appropriate values